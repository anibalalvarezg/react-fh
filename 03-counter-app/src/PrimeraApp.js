import React from 'react';

export const PrimeraApp = ({saludo, subtitulo}) => {
    return (
        <>
            <h1>{saludo}</h1>
            <p>{subtitulo}</p>
        </>
    );
};