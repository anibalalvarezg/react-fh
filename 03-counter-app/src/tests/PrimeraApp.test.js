import React from 'react'
import '@testing-library/jest-dom';
import { PrimeraApp } from '../PrimeraApp';
import { shallow } from 'enzyme';

describe('Test PrimeraApp component', () => {
    // test('should show message "Hola mundo"', () => {
    //     const greeting = "Hola mundo"; 

    //     const { getByText } = render(<PrimeraApp saludo={greeting} subtitulo="Test" />);

    //     expect(getByText(greeting)).toBeInTheDocument(); 
    // }); 

    test('should show PrimeraApp correctly', () => {
        const greeting = 'Hola mundo';
        const wrapper = shallow( <PrimeraApp saludo={greeting} subtitulo="Esto es una prueba" />);

        expect( wrapper ).toMatchSnapshot()
    })
    
});