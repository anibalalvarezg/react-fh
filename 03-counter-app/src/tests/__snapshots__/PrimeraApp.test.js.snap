// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test PrimeraApp component should show PrimeraApp correctly 1`] = `
<Fragment>
  <h1>
    Hola mundo
  </h1>
  <p>
    Esto es una prueba
  </p>
</Fragment>
`;
